### ISSUE TITLE
Resnet Vision model seems random

### ISSUE THREAD

**Issue by tdh512194** [RyanWangZf/MedCLIP#38]
the VIT model weight is reasonable but the Resnet weight outputs very random results, and always negative logits. Is the provided resnet weight correct?

**Comment by XNLHZ** [comment]
同样的问题

**Comment by PeirongLiL** [comment]
事实上，VIT和RESNET的权重都是随机的。只是RESNET的是负数，更为明显。ISSUE中起码十个人遇到了相同的问题，希望作者能帮忙答复。
